index.d.ts

```diff
@@ -60,22 +61,13 @@ declare namespace LightMyRequest {
     cookies?: { [k: string]: string }
   }
 
-  interface Request extends stream.Readable {
-    url: string
-    httpVersion: string
-    method: HTTPMethods
-    headers: http.IncomingHttpHeaders
-    rawHeaders: string[]
-    prepare: (next: () => void) => void
-    // @deprecated
-    connection: object
-  }
+  interface Request extends http.IncomingMessage {}
 
-  interface ServerResponse extends http.ServerResponse {} 
+  interface ServerResponse extends http.ServerResponse {}
```

Comment:
I'm not sure what's the real internal name of this object that light-my-request has.

Reply:
I would just drop this interface.